system/API/ResponseTrait.php

```diff
@@ -158,8 +158,8 @@ public function fail($messages, int $status = 400, string $code = null, string $
     /**
      * Used after successfully creating a new resource.
      *
-     * @param mixed  $data    Data.
-     * @param string $message Message.
+     * @param mixed  $data    data
+     * @param string $message message
```

Comment:
These seem silly. Is there perhaps a rule for "delete useless comments", like when `comment == property name`?

Reply:
I don't think there is a single fixer for this. But I think we can achieve this by removing then re-adding the phpdocs (which php-cs-fixer has).